test/simpleGaMultiSigTest.js

```diff
@@ -0,0 +1,57 @@
+const { use, assert, expect } = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const { utils, wallets } = require('@aeternity/aeproject');
+
+use(chaiAsPromised);
+
+describe('SimpleGAMultiSig', () => {
+  let aeSdk;
+  let gaContract;
+
+  const gaAccount = wallets[0];
+  const coSigner1 = wallets[1];
+  const coSigner2 = wallets[2];
+  const coSigner3 = wallets[3];
+  // get content of contract
+  const source = utils.getContractContent('./contracts/SimpleGAMultiSig.aes');
+
+  before(async () => {
+    aeSdk = await utils.getClient();
+
+    // attach the Generalized Account
+    await aeSdk.createGeneralizeAccount(
+        'authorize',
+        source,
+        ['2', `[${coSigner1.publicKey},${coSigner2.publicKey},${coSigner3.publicKey}]`],
```

Comment:
it looks so strange because we are still using HTTP compiler in sdk@10
https://github.com/aeternity/aepp-sdk-js/blob/ad36e7eaebd8b614e76cbda2c6ffd1f90cff217c/src/contract/ga/index.js#L95